lib/broccoli/ember-app.js

```diff
@@ -1341,15 +1321,17 @@ EmberApp.prototype.styles = function() {
     }.bind(this));
   }
 
-  this.import('vendor/addons.css');
-
   var vendorStyles = [];
   for (var outputFile in this._styleOutputFiles) {
+    var isMainVendorFile = outputFile === this.options.outputPaths.vendor.css;
     var headerFiles = this._styleOutputFiles[outputFile];
+    var inputFiles = isMainVendorFile ? ['modules/**/*.css'] : [];
```

Comment:
See https://github.com/ember-cli/ember-cli/blob/master/lib/broccoli/ember-app.js#L920-L951, which shows that this is the current semantics and not regressed by this PR.